### PR TITLE
interace: make game installation work even if game is not in the root zip

### PR DESCRIPTION
**Pull request by EXtremeExploit**

### CODE REVIEW

File: vita3k/interface.cpp [hunk 1]
```diff
@@ -94,27 +94,40 @@ bool install_archive(HostState &host, GuiState *gui, const fs::path &path) {
     }
 
     int num_files = mz_zip_reader_get_num_files(zip.get());
-    fs::path sfo_path = "sce_sys/param.sfo";
```
Comment (Zangetsu38) [inline vita3k/interface.cpp:97]: You can keep it

File: vita3k/interface.cpp [hunk 2]
```diff
@@ -96,25 +96,39 @@ bool install_archive(HostState &host, GuiState *gui, const fs::path &path) {
     int num_files = mz_zip_reader_get_num_files(zip.get());
     fs::path sfo_path = "sce_sys/param.sfo";
     bool theme = false;
+    std::string extra_path = "";
```
Comment (sunho) [inline vita3k/interface.cpp:99]: std::string extra_path; will work too :) just fyi

File: vita3k/interface.cpp [hunk 3]
```diff
@@ -96,25 +96,39 @@ bool install_archive(HostState &host, GuiState *gui, const fs::path &path) {
...
 
     for (int i = 0; i < num_files; i++) {
         mz_zip_archive_file_stat file_stat;
-        if (!mz_zip_reader_file_stat(zip.get(), i, &file_stat)) {
+        if (!mz_zip_reader_file_stat(zip.get(), i, &file_stat))
             continue;
-        }
-        if (fs::path(file_stat.m_filename) == "sce_module/steroid.suprx") {
+
+        if (fs::path(file_stat.m_filename).string().find("sce_module/steroid.suprx") != std::string::npos) {
```
Comment (sunho) [inline vita3k/interface.cpp:106]: can you just make fs::path(file_stat.m_filename).string() variable and use it below?

File: vita3k/interface.cpp [hunk 4]
```diff
@@ -96,25 +96,39 @@ bool install_archive(HostState &host, GuiState *gui, const fs::path &path) {
...
             LOG_CRITICAL("A Vitamin dump was detected, aborting installation...");
             fclose(vpk_fp);
             return false;
         }
-        if (fs::path(file_stat.m_filename) == "theme.xml")
+
+        if (fs::path(file_stat.m_filename).string().find("theme.xml") != std::string::npos)
```
Comment (sunho) [inline vita3k/interface.cpp:112]: here would be then if (variable.find("theme.xml") != std::string::npos)

**Pull request merged**